### DISCUSSION TITLE
Default CPU Boost Settings for Zephyrus G16 2024

### DISCUSSION THREAD

**Discussion by blackgold9** [seerge/g-helper#2418] [Q&A]
Hi folks, 
I got G-Helper running on my Zephyrus G16 2024 and started messing with the CPU boost settings trying to create a custom profile. I then realized that I'd messed with the boost settings for the default settings, and I no longer know what they're supposed to be. Does anyone know what the CPU boost settings are by default? Thanks!

**Comment by seerge** [comment]
@blackgold9 there is no default cpu boost setting, g-helper just reads what is set in your power plan currently and shows that by default.

In windows default is Enabled as far as I remember

**Reply by blackgold9** [reply]
Right, and I'm trying to get back to what those were before i messed with them.

P.S. Thanks for your work on this project

**Reply by IceStormNG** [reply]
The default from Windows is, afaik, Aggressive. But: only few CPUs are actually affected by this settings. Most Intel CPUs do not care about this state, as the turbo boost is fully hardware controlled and the CPU decides this on its own based on the EPP and EPB values.
